src/Concordium/Client/Runner.hs

```diff
@@ -3849,10 +3843,10 @@ processTransaction ::
   => BSL.ByteString
   -> Int
   -> ClientMonad m Types.BareBlockItem
-processTransaction source networkId =
+processTransaction source _networkId =
   case AE.eitherDecode source of
     Left err -> fail $ "Error decoding JSON: " ++ err
-    Right t  -> processTransaction_ t networkId True
+    Right t  -> processTransaction_ t _networkId True
```

Comment:
See comments in the parent PR related to this naming/removing the parameter.

Reply:
I added a TODO related to this.